src/Diagnostics.ModelsAndUtils/Models/Category.cs

```diff
@@ -18,7 +18,41 @@ public static Category Custom(string name)
             return new Category(name);
         }
 
+        // The predefined categories below are broken out by resource type, 
+        // but it does not mean they can only by used by that resource type.
+
+        // General
         public static readonly Category AvailabilityAndPerformance = new Category("Availability and Performance");
         public static readonly Category ConfigurationAndManagement = new Category("Configuration and Management");
+        public static readonly Category Deployment = new Category("Deployment");
+        public static readonly Category OpenSourceTechnologies = new Category("Open Source Technologies");
+        public static readonly Category SSL = new Category("SSL");
+
+        // Site Specific
+        public static readonly Category ProblemsWithAse = new Category("Problems with ASE"); // For site on ASE
+        public static readonly Category ProblemsWithWebjobs = new Category("Problems with Webjobs");
+
+        // Linux Specific
+        public static readonly Category DockerContainers = new Category("DockerContainers");
+
+        // ASE Specific
+        public static readonly Category Networking = new Category("Problems with Webjobs");
```

Comment:
Add the following:

-Network Configuration
-ASE Deployment
-Scaling
-Management